package.json

```diff
@@ -10,49 +10,60 @@
     "datastore"
   ],
   "author": "Juan Picado <juanpicado19@gmail.com>",
+  "contributors": [{
+    "name": "Ed Clement",
+    "email": "ed.clement@devintent.com",
+    "url": "https://github.com/edclement"
+  }],
   "license": "MIT",
   "homepage": "https://verdaccio.org",
   "repository": {
     "type": "git",
-    "url": "https://github.com/verdaccio/monorepo",
-    "directory": "plugins/google-cloud"
+    "url": "https://github.com/xlts-dev/verdaccio-google-cloud-storage"
   },
   "bugs": {
-    "url": "https://github.com/verdaccio/monorepo/issues"
+    "url": "https://github.com/xlts-dev/verdaccio-google-cloud-storage/issues"
   },
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "files": [
     "lib/"
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">=12"
   },
   "dependencies": {
-    "@google-cloud/datastore": "4.5.0",
-    "@google-cloud/storage": "3.5.0",
-    "@verdaccio/commons-api": "workspace:10.0.1",
-    "@verdaccio/streams": "workspace:10.0.0"
+    "@google-cloud/datastore": "6.5.0",
+    "@google-cloud/secret-manager": "3.10.1",
+    "@google-cloud/storage": "5.15.0",
+    "@verdaccio/commons-api": "10.0.1",
+    "@verdaccio/streams": "10.0.0"
   },
   "devDependencies": {
-    "@verdaccio/types": "workspace:10.0.0",
-    "memory-fs": "0.5.0"
+    "@types/jest": "26.0.24",
+    "@types/node": "14.14.31",
+    "@verdaccio/babel-preset": "10.0.0",
+    "@verdaccio/eslint-config": "10.0.0",
+    "@verdaccio/types": "10.0.0",
+    "babel-eslint": "10.1.0",
+    "cross-env": "7.0.3",
+    "eslint": "6.8.0",
+    "jest": "26.0.1",
+    "lint-staged": "9.5.0",
+    "prettier": "1.19.1",
```

Comment:
This is a pretty old version. Can you please open an issue to update prettier to a newer release? It's better to do it in a separate PR to keep this PR lean.

Reply:
opened #7